src/database/GameLoader.ts

```diff
@@ -66,8 +66,22 @@ export class GameLoader implements IGameLoader {
     });
   }
 
+  public async getByParticipantIdAsync(id: PlayerId | SpectatorId): Promise<Game> {
+    const d = await this.idsContainer.getGames();
+    const gameId = d.participantIds.get(id);
+    if (gameId === undefined) {
+      return Promise.reject(new Error('Not found'));
```

Comment:
```suggestion
      throw new Error('Not found');
```

In an `async` function can `throw`. No need to return a `Promise`.